quodlibet/po/fr.po

```diff
@@ -5661,156 +5622,147 @@ msgid "Display remaining time"
 msgstr "Montrer la durée restante"
 
 #: ../quodlibet/qltk/shortcuts.py:16
-#, fuzzy
 msgid "Main Window"
-msgstr "Paroles"
+msgstr "Fenêtre principale"
 
 #: ../quodlibet/qltk/shortcuts.py:17
 msgid "Seek backwards by 10 seconds"
-msgstr ""
+msgstr "Reculer de 10 secondes"
 
 #: ../quodlibet/qltk/shortcuts.py:18
 msgid "Seek forward by 10 seconds"
-msgstr ""
+msgstr "Avancer de 10 secondes"
 
 #: ../quodlibet/qltk/shortcuts.py:19
-#, fuzzy
 msgid "Focus the search entry"
-msgstr "Focaliser le lecteur en fonction"
+msgstr "Focaliser sur la requête de recherche"
 
 #: ../quodlibet/qltk/shortcuts.py:22
-#, fuzzy
 msgid "Reset filters and jump to the playing song"
-msgstr "Se déplacer dans la chanson en cours de lecture"
+msgstr "Réinitialiser les filtres et aller au titre en cours de lecture"
 
 #: ../quodlibet/qltk/shortcuts.py:26
 msgid "Open the information window for the selected songs"
-msgstr ""
+msgstr "Ouvrir la fenêtre d'informations des titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:27
 msgid "Open the tag editor for the selected songs"
-msgstr ""
+msgstr "Ouvrir le gestionnaire d'étiquettes des titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:28
-#, fuzzy
 msgid "Queue the selected songs"
-msgstr "Impossible de supprimer les chansons"
+msgstr "Placer en file de lecture les titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:29
-#, fuzzy
 msgid "Delete the selected songs"
-msgstr "Impossible de supprimer les chansons"
+msgstr "Supprimer les titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:30
 msgid "Show the inline search entry"
-msgstr ""
+msgstr "Afficher le contenu de la recherche interne"
```

Comment:
I agree that untranslated is better than incorrect, but I'm not sure it needs fixing. It says : open a search dialog, implied, that searches inside currently focused area (songlist, treeview in filebrowser ..) 
"Afficher le contenu de la recherche interne" doesn't say that.
"Ouvre recherche intégrée/rapide" would be as much descriptive as "Open inline search dialog entry"
Could say something like "Ouvre la recherche rapide de/dans la liste des titres, ou l'arborescence du Navigateur de Fichiers .. "
Anyway, that's just a shortcut description here, and that shortcut is kind of useless anyway since you get the same result by just starting typing when songlist is focused, so no big loss here ;)